PageSpeed_Api.php

```diff
@@ -1,128 +1,1898 @@
+					'details'      => $this->v( $data, array( 'lighthouseResult', 'audits', 'resource-summary', 'details', 'items' ) ),
+					'instruction'  => __(
+						'TBD',
+						'w3-total-cache'
+					),
+				),
+				'largest-contentful-paint-element' => array(
+					'title'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'largest-contentful-paint-element', 'title' ) ),
+					'description'  => $this->v( $data, array( 'lighthouseResult', 'audits', 'largest-contentful-paint-element', 'description' ) ),
+					'score'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'largest-contentful-paint-element', 'score' ) ),
+					'displayValue' => $this->v( $data, array( 'lighthouseResult', 'audits', 'largest-contentful-paint-element', 'displayValue' ) ),
+					'details'      => $this->v( $data, array( 'lighthouseResult', 'audits', 'largest-contentful-paint-element', 'details', 'items' ) ),
+					'type'         => array(
+						'LCP',
+					),
+					'instruction'  => wp_kses(
+						sprintf(
+							// translators: 1 two HTML line break tags,
+							// translators: 2 HTML line break tag followed by opening HTML ul tag followed by opening HTML li tag,
+							// translators: 3 closing HTML li tag followed by opening HTML li tag, 4 closing HTML li tag followed by opening HTML li tag,
+							// translators: 5 closing HTML li tag followed by opening HTML li tag, 6 closing HTML li tag followed by opening HTML li tag,
+							// translators: 7 closing HTML li tag followed by closing HTML ul tag followed by two HTML line break tags,
+							// translators: 8 HTML line break tag followed by opening HTML ul tag followed by opening HTML li tag,
+							// translators: 9 closing HTML li tag followed by opening HTML li tag, 10 closing HTML li tag followed by opening HTML li tag,
+							// translators: 11 closing HTML li tag followed by opening HTML li tag, 12 closing HTML li tag followed by opening HTML li tag,
+							// translators: 13 closing HTML li tag followed by opening HTML li tag,
+							// translators: 14 closing HTML li tag followed by closing HTML ul tag followed by two HTML line break tags,
+							// translators: 15 HTML line break tag followed by opening HTML ul tag followed by opening HTML li tag,
+							// translators: 16 closing HTML li tag followed by opening HTML li tag, 17 closing HTML li tag followed by opening HTML li tag,
+							// translators: 18 closing HTML li tag followed by opening HTML li tag, 19 closing HTML li tag followed by opening HTML li tag,
+							// translators: 20 closing HTML li tag followed by closing HTML ul tag followed by two HTML line break tags,
+							// translators: 21 HTML line break tag followed by opening HTML ul tag followed by opening HTML li tag,
+							// translators: 22 closing HTML li tag followed by opening HTML li tag, 23 closing HTML li tag followed by closing HTML ul tag.
+							__(
```

Comment:
This is way too difficult for a translator to use... They should just translate strings and not have to worry so much about all the markup.